apis/storage/v1alpha1/backupstorage_types.go

```diff
@@ -136,6 +136,7 @@ const (
 	TypeRepositorySynced          = "RepositorySynced"
 	ReasonRepositorySyncSucceeded = "RepositorySyncSucceeded"
 	ReasonRepositorySyncFailed    = "RepositorySyncFailed"
+	TypeBackendSecretFound        = "TypeBackendSecretFound"
```

Comment:
We need to add reason for secret found or not.

Reply:
- Add a line break
- "TypeBackendSecretFound" -> "BackendSecretFound"